source/_docs/migrate-local.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Migrate Sites to Pantheon: From Your Local Machine
+description: Learn how to migrate locally hosted Drupal or WordPress site to Pantheon
+tags: [migrate, local]
+categories: []
+---
+
+This page discusses sites developed from your local machine, ready to deploy on Pantheon.
+
+We recommend using [Kalabox](/docs/kalabox) because of its integrated features to the platform. For large teams/sites, we recommend using [Multidev](/docs/multidev) instead of local development.
+Read more on what will only work for [Local Development](/docs/local-development/)
+
+If you're site is existing or can be synced only, use the [Guided Migration method](/docs/migrate). For cases where the guided migration will not work, follow the [manual migration method](/docs/migrate-manual).
```

Comment:
@eabquina what do you mean by:

> ...is existing or can be synced only...

Reply:
@alexfornuto the sentence is totally wrong and I've entered a change for that. This is meant to ask if its hosted somewhere else